src/status_im2/contexts/communities/discover/events.cljs

```diff
@@ -1,19 +1,42 @@
 (ns status-im2.contexts.communities.discover.events
-  (:require [camel-snake-kebab.core :as csk]
-            [clojure.string :as string]
-            [taoensso.timbre :as log]
+  (:require [taoensso.timbre :as log]
             [utils.re-frame :as rf]))
 
+(def commmunity-keys-renamed
```

Comment:
I had the impression we generally want to move data model transformations to `data_store/*` files, so that event handlers don't need to deal with the responsibility of knowing how the data arrives from status-go.

Feel free to ignore my feedback if there's more urgency to merge this PR. I realize now that this is not a clear agreement within the team. We have a mix of everything when it comes to the data model layer.